lib/core/index.tsx

```diff
@@ -11,6 +12,11 @@ export type EntryProps = ComponentBase & {
   network: Networks;
 };
 
+export type SideProps = ComponentBase & {
+  open: number;
+  minimised: number;
```

Comment:
```suggestion
  open: boolean;
  minimised: boolean;
```
These can be changed to booleans.

Reply:
this will require the whole dashboard to change both of the types though, no?